components/openindiana/time-slider/Makefile

```diff
@@ -21,7 +21,7 @@ COMPONENT_VERSION=	0.2.101
 COMPONENT_LICENSE=	CDDL
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count).1
```

Comment:
It is. We change COMPONENT_REVISION to achieve 2 goals:
1) clearly distinguish different package versions (at least for diagnostic purposes if not for fixing with userland-incorporation),
2) always trigger rebuild on component change.